modules/tinymce/src/core/test/ts/browser/caret/FakeCaretTypeTest.ts

```diff
@@ -0,0 +1,30 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { TinyHooks, TinySelections, TinyAssertions, TinyContentActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import TablePlugin from 'tinymce/plugins/table/Plugin';
+
+describe('browser.tinymce.core.FakeCaretTypeTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    base_url: '/project/tinymce/js/tinymce',
+    indent: false
+  }, [ TablePlugin ]);
```

Comment:
Is there a reason we include the Table plugin module here but don't have it in the config?

Reply:
Oops. I copied the setup from `FirefoxFakeCaretBeforeTableTypeTest` and forgot to remove the table plugin.

Reply:
I figured it was probably something simple like that 😄